TombEngine/Scripting/Internal/TEN/Misc/Miscellanous.cpp

```diff
@@ -180,6 +182,12 @@ namespace Misc
 		TrInput &= ~(1 << actionIndex);
 	}
 
+	static void KeyClearInventory()
```

Comment:
What is the purpose of this command? You have hardcoded action index `13` here.

Reply:
action index 13 clears 	TrInput &= ~(1 << actionIndex);
but Inventory menu uses DbInput

Reply:
I mean, do we need specific function which clears only inventory key? We can just modify `KeyClear` script function, so it also clears `DbInput` and inventory is never called.

Reply:
aah,you are right. I removed the unnecessary ClearMenuKey function and added the DbInput &= ~(1 << actionIndex); into the KeyClear() function. :)